### PR TITLE
exclude from Datadog APM

### PR DESCRIPTION
**Pull request by robatwave**
Reference: https://docs.datadoghq.com/tracing/guide/trace_sampling_and_storage/?tab=python

### CODE REVIEW

**Comment by mwarkentin** [comment]
Awesome, thanks! Let me know if you need anything to test this (eg. dev release or whatever).

**Comment by robatwave** [comment]
@mwarkentin yeah a dev release would probably be easiest

**Comment by mwarkentin** [comment]
Cool, ping me tomorrow and I’ll get that pushed for you.

**Comment by mwarkentin** [comment]
@robatwave I've pushed a dev release: https://pypi.org/project/django-watchman/#history

File: watchman/views.py [hunk 1]
```diff
@@ -41,6 +41,13 @@ def _disable_apm():
         newrelic.agent.ignore_transaction(flag=True)
     except ImportError:
         pass
+    # Datadog
+    try:
+        from ddtrace import tracer
+        from ddtrace.constants import MANUAL_DROP_KEY
+        tracer.current_span().set_tag(MANUAL_DROP_KEY)
```
Comment (JBKahn) [inline watchman/views.py]: `current_span` could return `None` so catch the `AttributeError` as well.

**Comment by robatwave** [comment]
@mwarkentin one more dev release please

**Comment by mwarkentin** [comment]
@robatwave pushed `dev2`

**Comment by mwarkentin** [comment]
@robatwave can you:

* start a 0.18 section for https://github.com/mwarkentin/django-watchman/blob/master/HISTORY.rst
* Add datadog to the APM docs: https://github.com/mwarkentin/django-watchman/blob/master/README.rst#apm-ie-new-relic

**Comment by ryanwilsonperkin** [comment]
Confirmed on my app!

**Pull request merged**